Greet-Dough/src/main/java/store/PostStore.java

```diff
@@ -0,0 +1,14 @@
+package store;
+
+import model.Post;
+import utility.TrackerID;
+
+public interface PostStore extends TrackerID {
```

Comment:
if for some reason each store has its own way to assign ids, it probably makes sense to allow each store to assign uuid/int/etc. i would maybe opt to keep `public interface PostStore` with no extended functionality, and then you can do something like `public class PostStoreInMemoryImpl extends TrackerId implements PostStore` since it's bound to the implementation and not so much to the interface definition